pkg/gadgets/networkpolicy/gadget.go

```diff
@@ -182,7 +176,6 @@ func (f *Trace) Stop(trace *gadgetv1alpha1.Trace) {
 	output := f.out.String()
 	log.Infof("Network Policy Advisor output:\n%s\n", output)
 
-	trace.Status.OperationError = ""
```

Comment:
Out of scope for this PR but I think here we are missing to set the `State` to `Stopped`.

Reply:
Done in this PR so we don't forget about it.